cxx-squid/src/main/java/org/sonar/cxx/visitors/CxxPublicApiVisitor.java

```diff
@@ -65,8 +65,7 @@
 public class CxxPublicApiVisitor<GRAMMAR extends Grammar> extends
   AbstractCxxPublicApiVisitor<Grammar> {
 
-  private static final Logger LOG = LoggerFactory
-    .getLogger("CxxPublicApiVisitor");
+  private static final Logger LOG = Loggers.get("CxxPublicApiVisitor");
```

Comment:
@jmecosta others are using e.g. ´private static final Logger LOG = Loggers.get(UTCoverageSensor.class);` instead of string as parameter?